### DISCUSSION TITLE
Failed to fetch 7TV

### DISCUSSION THREAD

**Discussion by moadAlami** [SevenTV/chatterino7#58] [Q&A] [answered]
Whenever I start or refresh chatterino I get this error.
`Failed to fetch 7TV channel emotes. (unknown error)`
I'm using the 7.3.4 build on Windows 10. The same build is working fine on another pc tested both on windows 10 and archlinux.

**Comment by zneix** [comment, marked as answer]
This might've been a temporary issue with the 7tv servers, or you might've got blocked from its API. Should be resolved now. Please comment below if it still persists.
Alternatively, please check your connection to `api.7tv.app` domain and see if you can even reach it / resolve through DNS.

**Reply by moadAlami** [reply]
Turns out it was blocked by the establishment I was at.
